### PR TITLE
Added test roles to support dnos6_command module

### PR DESCRIPTION
**Pull request by skg-net**
Added test roles to support dnos6_command module.

### CODE REVIEW

File: dnos6.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
```
Comment (gundalow) [inline dnos6.yaml]: This play should only run on dnos6 machines, so can you please set this to:

`hosts: dnos6`

**Comment by gundalow** [comment]
This, and the other `dnos*`plays need adding to `all.yaml`

**Comment by skg-net** [comment]
ready_for_review

**Comment by gundalow** [comment]
@skg-net Can you please rebase then I'll merge it. Thanks :)

**Comment by skg-net** [comment]
Rebased, please check it out.

**Comment by gundalow** [comment]
Merged, thanks :)

**Pull request merged**